file-formats/aifn/type/zif_aff_aifn_v1.intf.abap

```diff
@@ -0,0 +1,27 @@
+INTERFACE zif_aff_aifn_v1
+  PUBLIC.
+  TYPES:
+    "! <p class="shorttext">Namespace Details</p>
+    "! Namespace details
+    BEGIN OF ty_namespace_details,
+      "! <p class="shorttext">AIF Namespace</p>
+      "! AIF Namespace
```

Comment:
```suggestion
      "! AIF namespace
```

Please use sentence case in descriptions

Reply:
thanks, description was adapted